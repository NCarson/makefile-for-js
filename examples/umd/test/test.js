
DIR = '../pkg'
const lib = require(DIR + '/index')
console.log(lib)
const foo = require(DIR + '/lib/Foo')
console.log(foo)
const bar = require(DIR + '/lib/Bar')
console.log(bar)

