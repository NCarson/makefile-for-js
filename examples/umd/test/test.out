{ Foo: 3, Bar: 2 }
3
2
